lib/plugins/features/movies/controller.js

```diff
@@ -12,9 +12,20 @@ exports.create = (payload) => {
 };
 
 exports.get = (query) => {
```

Comment:
Nit: `get` is a bit ambiguous because there can be multiple endpoints with HTTP method `GET`. The vernacular we usually use is:

- `POST /movies` - `create`
- `GET /movies/{id}` - `retrieve`
- `GET /movies` - `list`
- `DELETE /movies` - `delete`